### ISSUE TITLE
NanostackRfPhyAtmel.cpp warnings

### ISSUE THREAD

**Issue by teetak01** [ARMmbed/atmel-rf-driver#62]
### Description
Mbed OS 5.9.1 (and earlier)
IAR ANSI C/C++ Compiler V7.80.1.28/LNX for ARM

```
08:50:40 [Warning] NanostackRfPhyAtmel.cpp@386,14: [Pe177]: function "rf_tx_power_set" was declared but never referenced
08:50:40 [Warning] NanostackRfPhyAtmel.cpp@1377,29: [Pe177]: function "rf_ack_wait_timer_stop" was declared but never referenced
08:50:40 [Warning] NanostackRfPhyAtmel.cpp@2011,28: [Pe177]: function "rf_tx_power_get" was declared but never referenced
08:50:40 [Warning] NanostackRfPhyAtmel.cpp@2023,0: [Pe177]: function "rf_enable_antenna_diversity" was declared but never referenced
```

**Comment by ciarmcom** [comment]
ARM Internal Ref: ONME-3725

**Comment by SeppoTakalo** [comment]
Fixed in #63
